chanzuckerberg/single-cell-data-portal
# feat(cell-cards): Cell Cards Landing Page

## Reason for Change

- #4497

Boiler plate PR for chopping up Cell Cards work into pieces. Subsequent PRs will involve adding more components and features on top of this PR

## Changes

- add
    - Boiler plate for Cell Cards view
- remove
- modify

## Testing steps

- Navigate to https://localhost:3000/cell-cards
- Page should load with initial text
- Navigate to https://localhost:3000/cell-cards/CL_2000043
- Page should load and be empty

## Notes for Reviewer
